docs/Database-Setup.md

```diff
@@ -2,7 +2,25 @@ You can setup the database manually or using the automated script.
 
 ## How to setup the AzerothCore DB using the automated script
 
-### Edit script configuration
+### Use the default configurations:
+
+The `db_assembler` script and the `acore.sh` dashboard both use by default the **acore** db user for the database connection. Running for the first time the assembler command should create the user for you, however, if you're having permissions error or any other issue with that user, please run the following queries in your database with the root user:
```

Comment:
let's make this a bit more user-friendly and show how to connect to MySQL and run the commands